sdk/logs/src/test/java/io/opentelemetry/sdk/logs/SdkLogEmitterProviderTest.java

```diff
@@ -28,15 +30,19 @@
 @MockitoSettings(strictness = Strictness.LENIENT)
 class SdkLogEmitterProviderTest {
 
+  private static final long NOW = TimeUnit.MILLISECONDS.toNanos(System.currentTimeMillis());
   @Mock private LogProcessor logProcessor;
+  @Mock private Clock clock;
```

Comment:
It doesn't look like this mock clock is needed for this test.

Reply:
I thought it would be easy coverage, but I will move it to a dedicated test method.